src/Auxillary/json_schema/include/make_schema.hpp

```diff
@@ -28,6 +28,23 @@ namespace Aux::schema {
    */
   json make_boundary_schema(int num_values);
 
+  /**
+   * @brief Helper to create an initial value configuration json schema for
+   * a component (this is then assembled into a larger initial value json)
+   *
+   * @param num_interpol_points The number of interpolation points (minimum &
+   * maximum) if this number is <=0, this is interpreted as no restriction
+   * @param num_values The number of values at one interpolation point
+   * @param contains_interpol_point_schemas Json schemas describing properties
+   * of interpolation points which at least one of the interpolation points
+   * have to satisfy. (e.g. {"minimum": 0} means that at least one of the
+   * interpolation points has to be larger 0)
+   * @return json
+   */
```

Comment:
Shouldn't minimum mean that no value is allowed to be smaller than 0?

Reply:
yes if it was a normal schema. It is a `contains_...` schema though. The `idx`th Schema from this vector of `contains_...` schemas is pasted into an `"allOf": [idx]{"contains":$schema}` block describing the list of points. In particual every one of these schemas describes only one of the points in the list.